### PR TITLE
disable grafana's outgoing traffic

### PR DESCRIPTION
**Pull request by divolgin**

### CODE REVIEW

File: addons/prometheus/0.33.0/grafana/grafana-config.yaml [hunk 1]
```diff
@@ -0,0 +1,537 @@
...
+    # If enabled and user is not anonymous, data proxy will add X-Grafana-User header with username into the request, default is false.
+    ;send_user_header = false
+    
+    #################################### Analytics ####################################
+    [analytics]
+    # Server reporting, sends usage counters to stats.grafana.org every 24 hours.
+    # No ip addresses are being tracked, only simple counters to track
+    # running instances, dashboard and error counts. It is very helpful to us.
+    # Change this option to false to disable reporting.
+    reporting_enabled = false
```
Comment (divolgin) [inline addons/prometheus/0.33.0/grafana/grafana-config.yaml:137]: this changed

File: addons/prometheus/0.33.0/grafana/grafana-config.yaml [hunk 2]
```diff
@@ -0,0 +1,537 @@
...
+    
+    # Set to false to disable all checks to https://grafana.net
+    # for new vesions (grafana itself and plugins), check is used
+    # in some UI views to notify that grafana or plugin update exists
+    # This option does not cause any auto updates, nor send any information
+    # only a GET request to http://grafana.com to get latest versions
+    check_for_updates = false
```
Comment (divolgin) [inline addons/prometheus/0.33.0/grafana/grafana-config.yaml:144]: this changed

File: addons/prometheus/0.33.0/grafana/grafana-config.yaml [hunk 3]
```diff
@@ -0,0 +1,537 @@
...
+    ;admin_user = admin
+    
+    # default admin password, can be changed before first start of grafana,  or in profile settings
+    ;admin_password = admin
+    
+    # used for signing
+    ;secret_key = SW2YcwTIb9zpOOhoPsMm
+    
+    # disable gravatar profile images
+    disable_gravatar = true
```
Comment (divolgin) [inline addons/prometheus/0.33.0/grafana/grafana-config.yaml:164]: this changed

File: addons/prometheus/0.33.0/grafana/grafana-config.yaml [hunk 4]
```diff
@@ -0,0 +1,537 @@
...
+    # set cookie SameSite attribute. defaults to `lax`. can be set to "lax", "strict" and "none"
+    ;cookie_samesite = lax
+    
+    # set to true if you want to allow browsers to render Grafana in a <frame>, <iframe>, <embed> or <object>. default is false.
+    ;allow_embedding = false
+    
+    #################################### Snapshots ###########################
+    [snapshots]
+    # snapshot sharing options
+    external_enabled = false
```
Comment (divolgin) [inline addons/prometheus/0.33.0/grafana/grafana-config.yaml:184]: this changed

**Review by areed — APPROVED** [review]

**Pull request merged**